postgres/osmap.yaml

```diff
@@ -19,6 +19,17 @@ Debian:
   pkg_dev: postgresql-server-dev-all
   pkg_libpq_dev: libpq-dev
 
+  pkg: 'postgresql-{{ repo.version }}'
+  pkg_client: 'postgresql-client-{{ repo.version }}'
+  pkgs_extra: [ postgresql-contrib, postgresql-plpython-{{ repo.version }}, libpostgresql-jdbc-java]
```

Comment:
Partly agree.  I think "pkgs_extra = []" should be default in pillar.example uncommented.   This provides testing advantage since we can test/support 'pkgs_extra' parameter via osmap.yaml definitions by simply removing from pillars. It should be possible to test 'pkg_extras' feature works for any distribution without forcing it.  Let me know what you think.

Reply:
I added `pkgs_extra: []` to pillar.example.

Reply:
My point is that the formula by default should provide minimum working installation. And in many use cases a user just don't need to wait and download a half of the distribution just to get a separate DB for WordPress backend or something.
I understand that those packages are system-specific, but still I think it is up to the user to figure out and fill in any additional stuff. I suggest to keep `pkgs_extra: []` as default and mention it in `pillar.example` without exposing values for some particular distro.